### ISSUE TITLE
Discussion on Nitrogen leaching

### ISSUE THREAD

**Issue by sarahcleary** [APSIMInitiative/ReferencePanel#23]
Action from Feb RP Strategy Meeting
Enli and Hamish to discuss synergies in Nitrogen leaching work

**Comment by sarahcleary** [comment]
Any update - @HamishBrownPFR / @EnliWang / @sno036 ?

**Comment by sarahcleary** [comment]
Any update - @HamishBrownPFR / @EnliWang / @sno036 ?

**Comment by HamishBrownPFR** [comment]
Discussed with Neil, Enli, Hamish.
Need for an effort focused on pulling together validation of solute leaching in APSIM
Lots of sugar cane date, Iowa maize data, Lochia data, Mora, New Zealand N leaching data, China data (Enli), chloride and bromide data.

**Comment by sarahcleary** [comment]
@Keith-Pembleton to draft an outline for a PhD project which could be advertised across the AI universities.

**Comment by sarahcleary** [comment]
Reminder - @Keith-Pembleton to draft an outline for a PhD project which could be advertised across the AI universities.

**Comment by sarahcleary** [comment]
@Keith-Pembleton - any update?

**Comment by sarahcleary** [comment]
@Keith-Pembleton - any update?

**Comment by sarahcleary** [comment]
@Keith-Pembleton - any update?

**Comment by sarahcleary** [comment]
@Keith-Pembleton  - can this issue be closed?

**Comment by EnliWang** [comment]
I have one colleague in China working on the measurement and modelling of N leaching. As mentioned earlier by Hamish, we still need an effort focused on pulling data together into APSIM.

**Comment by sarahcleary** [comment]
@EnliWang - any update?  or when would you like a follow up on this?

**Comment by sarahcleary** [comment]
@EnliWang and/or @HamishBrownPFR - any update?

**Comment by sarahcleary** [comment]
@EnliWang and/or @HamishBrownPFR - any update?  or close issue?

**Comment by sarahcleary** [comment]
@EnliWang - any update?

**Comment by sarahcleary** [comment]
@EnliWang and/or @HamishBrownPFR - any update? or close issue?

**Comment by sarahcleary** [comment]
@EnliWang and/or @HamishBrownPFR - any update? or close issue?

**Comment by HamishBrownPFR** [comment]
I think we need to keep this open and work out how we can resource someone to take the job on.

**Comment by Keith-Pembleton** [comment]
I could talk to the soils CRC re PhD scholarship but I will need to present them a potential student as part of those discussions.  If anyone has one in mind send me a email

**Comment by HamishBrownPFR** [comment]
Could we put a case to the SC to fund a Ph.D. student?  A better way to go might be to get the funding in place and then recruit a good student

**Comment by sno036** [comment]
TBH - I am not all that keen on the AI funding a PhD student for this - too high a likelihood that it not end up with a solution for this issue, Cofunding a postdoc or a research project with a specific deliverable then more likely.

**Comment by Keith-Pembleton** [comment]
Have spoken to my SC rep.  The longer term nature (ie greater than 12 months) may be an issue, but it was not a direct no.  Leverage with another funding source I guess would be favorable.  We could try to pitch it without a candidate student in mind the worst they can say is no.  

What path to go down (project/postdoc vs PhD) depends on how long we want to potentially wait.  PhD scholarships are relatively easier to get compared to project funding for a postdoc. But the risk in a PhD are greater. 

Risks in not achieving anything could be mitigated through careful design of the PhD project, candidate selection and resourcing software development in addition to the PhD to ensure what gets done makes it into release.

**Comment by HamishBrownPFR** [comment]
@sno036 I agree but was worried that the cost might be too great for the SC with a postdoc or as a research project.  Should we try to cost something up as a research project and test the water?

**Comment by sarchontoulis** [comment]
The N leaching topic is of very interest. My questions to catch up with this: 1) what are the specific issues to be addressed? 2) is N leaching in subsurface tile drainage considered?

**Comment by HamishBrownPFR** [comment]
@sarchontoulis I will give you my recollection of the issue and others are free to correct me.  There are lots of situations where N leaching predictions are important for APSIM users but there has not been an effort to collate a broad validation set for this as we do for specific crops.  The idea is to get some one (Ph.D? PostDoc? Scientist?) to pull together a test set for predictions of leaching and other N balance data so we can build confidence in APSIMs predictions, set a bench mark in the performance tests  and identify any issues that may need addressing.
There is an analogous issue for transpiration and drainage.

**Comment by sarchontoulis** [comment]
Thanks @HamishBrownPFR . All clear now and this makes a lot of sense. An alternative thought to make a start here is each organization or interested individual to volunteer and add one NG simulation (including the corresponding soil N measurements) and then the postdoc/student adds more as needed

**Comment by sarahcleary** [comment]
@sarchontoulis / @HamishBrownPFR - can I close this issue?
